datacube/utils/geometry/_base.py

```diff
@@ -1389,35 +1389,30 @@ def lonlat_bounds(geom: Geometry,
     if geom.crs.geographic:
         return geom.boundingbox
 
```

Comment:
These changes are not mine.  I think something went wrong with the rebase.  See also below.